lib/kitchen/note_element.rb

```diff
@@ -62,10 +58,26 @@ def self.is_the_element_class_for?(node)
 
     protected
 
-    def translation_key_in(possible_translation_keys)
-      keys = possible_translation_keys & classes
+    def key
+      if I18n.t(document.short_name).instance_of?(String)
```

Comment:
Now that we have book-specific translations, I think we can take `chemistry` out of the translation keys don't you think?  B/c if we have `/books/chemistry/locales/en.yml` those translations are for chemistry by definition.  So I think we can get away from lookup by `document.short_name`.  

```yaml
en: 
  note_titles:
    chemist-portrait: Portrait of a Chemist
```

Reply:
I think we have to keep using the `document.short_name` because of the case where the same class may be in use in multiple books, but mapping to a different title. (ex. I believe `media-2` is used in both u-physics and calculus for different titles.) What do you think?

Reply:
But in such a case those will be different recipe runs so no conflict?

Reply:
I get it now, works for me 👍